packages/@okta/vuepress-theme-prose/assets/css/components/_top-bar.scss

```diff
@@ -1,92 +1,263 @@
 $gray-900: #2f3f4a;
-.top-bar {
-  display: block;
-  background-color: cv('brand');
-  padding: 0rem;
-  z-index: 20;
-  border-bottom: none;
 
-  &--search {
-    &--mobile {
-      @include media(">=tablet") {
-        display: none;
-      }
+.top-bar-nav{
+  display: flex;
+  flex-direction: row;
+  padding: 30px;
+  align-items: center;
+  font-size: 14px;
+  height: 95px;
+  @include media("<=tablet") {
+    padding: 20px 30px;
+    height: 75px;
+    justify-content: space-between;
+  }
+  &--logo{
+    display: block;
+    -ms-flex-negative: 0;
+    flex-shrink: 0;
+    height: 32px;
+    position: relative;
+    z-index: 15;
+    svg {
+      fill: cv('gray', '000');
+      width: 91.54967px;
```

Comment:
nit: this looks like a calculated value - good to go with 91px?

Reply:
Again, copied from /blog. As I understood, main goal of that task was to make two headers looking same.